Calibration/HcalCalibAlgos/macros/CalibSort.C

```diff
@@ -418,6 +432,20 @@ void sort(std::vector<record>& records, bool debug) {
   }
 }
 
+void sortNew(std::vector<record>& records, bool debug) {
```

Comment:
Not sure I understand. Presumably you checked it and now the new one is validated? (It's essentially one line of code now since you use a standard function ) @bsunanda